docs/PAYLOAD.md

```diff
@@ -72,6 +72,121 @@ Some ways to handle this *double* event are:
 - send two pushes, one to be processed in the background, and the other to show up in the shade.
 - include a unique ID in your push so you can check to see if you've already processed this event.
 
+# Message Format Overview
+
+## Android Message Format
+
+The JSON push message can contain the following fields, see https://developers.google.com/cloud-messaging/http-server-ref for a complete list.
+
+```javascript
+var content = {
+  "priority": "normal", // Valid values are "normal" and "high."
+  "content_available": "0", // see "Background Notifications" below
+  "data": {
+    "title": "A short string describing the purpose of the notification",
+    "message": "The text of the alert message", // "body" can be used as alias, is converted to "message"
+    // localozation of message is possible
+    "count": 5, // set the badge notification count at app icon
+    "sound": "default", // play default sound ... or "soundname", see [Android Sound](#sound) section
+    "notId": 1, // unique ID for the message, used for grouping, see below
+    "custom_key1": "value1",
+    "custom_key2": "value2"
+  },
+  // "notification": { ... } // can be used instead of data for title and message, see ....
```

Comment:
Adding in the `notification` part of the payload will completely screw up the plugin as per [notification vs data payloads](https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/PAYLOAD.md#notification-vs-data-payloads). Don't even make a reference to it.